ofrak_patch_maker/ofrak_patch_maker_test/test_arm_toolchain.py

```diff
@@ -116,6 +120,12 @@ def test_hello_world(toolchain_under_test: ToolchainUnderTest):
     run_hello_world_test(toolchain_under_test)
 
 
+def test_relocatable(toolchain_under_test: ToolchainUnderTest):
+    # LLVM userspace test can't supply external symbols and LLVM non-userspace test can't link .got for some reason
+    if toolchain_under_test.toolchain != LLVM_12_0_1_Toolchain:
+        run_relocatable_test(toolchain_under_test)
```

Comment:
Instead of having this test pass, maybe we should [use `pytest.skip`](https://stackoverflow.com/questions/38442897/how-do-i-disable-a-test-using-pytest)? 

We seem to use it in one or two places in the code, though our usage differs from the suggestions in the linked StackOverflow post.

https://github.com/redballoonsecurity/ofrak/blob/5ca2c3f6bc8b80e647bfa4d9e81c175d6d0b14ee/ofrak_core/pytest_ofrak/patterns/register_usage_analyzer.py#L444-L454